src/app/containers/InlineLink/__snapshots__/index.test.jsx.snap

```diff
@@ -60,6 +60,36 @@ exports[`InlineLinkContainer external link accessibility should be explicitly ma
 </div>
 `;
 
+exports[`InlineLinkContainer internal link accessibility should contain an anchor 1`] = `
+.emotion-0 {
+  color: #222222;
+  border-bottom: 1px solid #B80000;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+}
+
+.emotion-0:visited {
+  color: #6E6E73;
+  border-bottom: 1px solid #6E6E73;
+}
+
+.emotion-0:focus,
+.emotion-0:hover {
+  border-bottom: 2px solid #B80000;
+  color: #B80000;
+}
+
+<div>
+  <a
+    aria-describedby="1234"
+    class="emotion-0 emotion-1"
+    href="/news/articles/c0g992jmmkko#1234"
+  >
```

Comment:
This snapshot does not show an `id` attribute in `<a>` tag allowing links to the page with `#` in them to jump to the desired link.

Just my personal opinion but in these situations I tend to prefer explicit tests that look for the presence of an `id` attribute rather than relying on snaphshots.

Reply:
I completely agree with you @andrewscfc. We need a test to check whether the link jumps to the correct id. This snapshot, however, ensures that the`#` renders along with the rest of path. Simorgh was getting rid of the `#` before this PR.

Reply:
I've added proper unit tests :)

Reply:
Can we remove this snapshot now?

Reply:
Done! Thank you

Reply:
Why do we need to remove the snapshot? It's useful to see the output